mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/InMemoryMessageId.java

```diff
@@ -77,8 +76,6 @@ public final boolean equals(Object obj) {
 
     @Override
     public String toString() {
-        return MoreObjects.toStringHelper(InMemoryMessageId.class)
-                .add("value", value)
-                .toString();
+        return serialize();
```

Comment:
Right, this commit will not be integrated, see 08f0532a7985830e3e31807a51e447594e5226f6